src/components/mapping/Mapping.tsx

```diff
@@ -0,0 +1,117 @@
+/* Import Dependencies */
+import { useEffect } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import classNames from 'classnames';
+import { Tabs, TabList, Tab, TabPanel } from 'react-tabs';
+import { Container, Row, Col } from 'react-bootstrap';
+
+/* Import Store */
+import { useAppSelector, useAppDispatch } from 'app/Hooks';
+import { getMapping, setMapping } from 'redux/mapping/MappingSlice';
+
+/* Import Utilities */
+import KeycloakService from 'keycloak/Keycloak';
+
+/* Import Components */
+import Header from 'components/Header/Header';
+import TitleBar from 'components/general/IDCard/titleBar/TitleBar';
+import IDCard from 'components/general/IDCard/IDCard';
+import MappingTab from './components/MappingTab';
+
+/* Import API */
+import GetMapping from 'api/mapping/GetMapping';
+
+
+const Mapping = () => {
+    /* Hooks */
+    const dispatch = useAppDispatch();
+    const params = useParams();
+    const navigate = useNavigate();
+
+    /* Base variables */
+    const mapping = useAppSelector(getMapping);
+
+    /* OnLoad: fetch Source System */
+    useEffect(() => {
+        const mappingId = `${params.prefix}/${params.suffix}`;
+
+        if (mapping?.id !== mappingId) {
+            GetMapping(mappingId).then((mapping) => {
+                if (mapping) {
+                    /* Set Source System */
```

Comment:
I think this should be /* Set Mapping */

Reply:
Fixed